RememberThese.txt

```diff
@@ -0,0 +1 @@
+mongoimport --jsonArray -d apatight -c restaurants2  --file places.json --numInsertionWorkers 8
```

Comment:
You can increase the # of workers to make it faster.  Try 16 and see if it makes a difference.  Not sure if the # of workers === # of cores but using 16 helped mine.

Reply:
Thanks, will try!